<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunConfigurationsDialog</class>
 <widget class="QDialog" name="RunConfigurationsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="QComboBox" name="comboBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GLS</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="comboBox_3">
     <item>
      <property name="text">
       <string>Strip Packing Problem</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Minimal Square Packing Problem</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Minimal Rectangle Packing Problem</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Single Knapsack Problem</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_6">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Cluster Time Fraction</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboBox_2">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>Current</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Random</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bottom Left</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Metaheuristic:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_7">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Zoom Search Ratio</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Max. worse iterations:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Packing Problem</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QTimeEdit" name="timeEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>10</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>hh:mm:ss</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSpinBox" name="spinBox">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Initial solution:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Use Zoomed Search Map</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Time limit:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Container lenght:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBox_4">
     <item>
      <property name="text">
       <string>Random</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bagpipe</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Rectangular Packing Method</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QSpinBox" name="spinBox_3">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RunConfigurationsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RunConfigurationsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_7</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
